src/aggregationserver/databaseclient.h

```diff
@@ -76,6 +76,14 @@ class DatabaseClient {
         const std::vector<std::string>& condition_values
     );
 
+    const pqxx::result GetMarkerInfo(
+        int experiment_run_id,
+        std::string start_time = "",
```

Comment:
maybe std::string start_time{""};
If you care about the strings being initializes, should you not also initialize the run_id?

Also 
const std::vector<std::string>& condition_columns{},